helm/kubernetes-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -74,4 +75,45 @@ text_color "$MAGENTA"
   assert_failure
 }
 {{- end }}
+
+@test "Conjur Account is valid" {
+  display_info "Validating account from the info endpoint for enterprise"
+
+  cmd=( curl -k --connect-timeout 5 "$conjurApplianceUrl"/info -o "$TEMP_INFO_FILE")
+  display_info "Running ${cmd[@]}"
+  run "${cmd[@]}"
```

Comment:
Not a big deal, but could we drop the `run` from this, i.e. just use this:
```
"${cmd[@]}"
```
since we're not using the `$status` nor `$output` from this command?

Reply:
done